docs/operator.md

```diff
@@ -2120,6 +2120,68 @@ The [Kubernetes memory requests :octicons-link-external-16:](https://kubernetes.
 | ----------- | ---------- |
 | :material-code-string: string     | `200m` |
 
+## <a name="operator-users-section"></a>Users section
+
+The `users` section in the [deploy/cr.yaml  :octicons-link-external-16:](https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml) file contains various configuration options [to configure custom MySQL users via the Custom Resource](users.md#create-users-in-the-custom-resource).
+
+### `users.name`
+
+The username of the MySQL user.
+
+| Value type | Example |
+| ---------- | ------- |
+| :material-code-string: string | `my-user` |
+
+### `users.dbs`
+
+Databases that the user authenticates against. If not specified, defaults to all databases (*). If user sets a administrative grants like SHUTDOWN, this field has to be omitted because administrative privileges are set on a global level.
+
+| Value type | Example |
+| ---------- | ------- |
+| :material-application-array-outline: array | <pre>- db1<br>-db2</pre> |
+
+### `users.hosts`
+
+Hosts that the users are supposed to connect from (if not specified, defaults to '%' - similar to what is happening in MySQL). 
+
+| Value type | Example |
+| ---------- | ------- |
+| :material-application-array-outline: array | `- localhost` |
+
+### `users.passwordSecretRef.name`
```

Comment:
Add that if the secret is not provided - the one will be autogenerated.

Reply:
done!

Reply:
Can you please add specifics?
 If not specified, user password will be generated and stored in a generated secret named {cluster-name}-custom-user-secret. The key that holds user password will be user name specified in spec.users.name.